src/app/header/header.component.html

```diff
@@ -1,6 +1,9 @@
 <div fxLayout="row" fxLayoutAlign="space-between" class="header-container">
   <div fxlayout="row" fxLayoutAlign="start center" fxLayoutGap="16px">
-    <span><i class="pi pi-bars "></i><i class="pi pi-angle-left"></i></span>
+    <span>
+      <i class="pi pi-bars "></i>
```

Comment:
Please take care of the unnecessary empty space
```suggestion
      <i class="pi pi-bars"></i>
```

Reply:
Resolved ...............Thank you